src/python/pants/jvm/dependency_inference/artifact_mapper_test.py

```diff
@@ -158,15 +158,19 @@ def test_third_party_dep_inference_resolve(rule_runner: RuleRunner) -> None:
         {
```

Comment:
This test is the motivation for this mad dash - look at how much more succint it is :D Great design @stuhood!